### PR TITLE
Quick notes about the (un)install/search commands for .NET6

### LINKED ISSUE #32949: dotnet install command.
From [this](https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-project-template) documentation page, `dotnet new install` does not seem to work on 6.0.111, But I noticed that the new format was able to work for me using the `dotnet new --install` stated in the docs [here](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-install)

Current dotnet versions. System is MacOS
```
.NET SDKs:
Version      Status
----------------------------------------
6.0.111      Up to date.
6.0.202      Patch 6.0.203 is available.

Try out the newest .NET SDK features with .NET 7.0.100.

.NET Runtimes:
Name                          Version      Status
---------------------------------------------------------------------
Microsoft.NETCore.App         6.0.1        Patch 6.0.11 is available.
Microsoft.AspNetCore.App      6.0.4        Patch 6.0.11 is available.
Microsoft.NETCore.App         6.0.4        Patch 6.0.11 is available.
Microsoft.AspNetCore.App      6.0.11       Up to date.
Microsoft.NETCore.App         6.0.11       Up to date.

```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 097102bb-f6d9-c469-5556-ae1f16fdc4fe
* Version Independent ID: 77b25381-6e0f-949f-5f96-b83e9b61e9fc
* Content: [Create a project template for dotnet new - .NET](https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-project-template)
* Content Source: [docs/core/tutorials/cli-templates-create-project-template.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/cli-templates-create-project-template.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

---
[Associated WorkItem - 52198](https://dev.azure.com/msft-skilling/Content/_workitems/edit/52198)

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

Added notes about the commands for .NET 6.

Fixes #32949

@vlada-shubina ready for review. Thanks!

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| _docs/core/tutorials/cli-templates-create-item-template.md_ | [Preview: docs/core/tutorials/cli-templates-create-item-template](https://review.learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-item-template?branch=pr-en-us-34546) |
| _docs/core/tutorials/cli-templates-create-project-template.md_ | [Preview: docs/core/tutorials/cli-templates-create-project-template](https://review.learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-project-template?branch=pr-en-us-34546) |
| _docs/core/tutorials/cli-templates-create-template-package.md_ | [Preview: docs/core/tutorials/cli-templates-create-template-package](https://review.learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-template-package?branch=pr-en-us-34546) |

### CODE REVIEW

File: docs/core/tutorials/cli-templates-create-template-package.md [hunk 1]
```diff
@@ -34,7 +34,10 @@ In this part of the series you'll learn how to:
...
 
-Template packages are represented by a NuGet package (_.nupkg_) file. And, like any NuGet package, you can upload the template package to a NuGet feed. The `dotnet new --install` command supports installing template package from a NuGet package feed. Additionally, you can install a template package from a _.nupkg_ file directly.
+Template packages are represented by a NuGet package (_.nupkg_) file. And, like any NuGet package, you can upload the template package to a NuGet feed. The `dotnet new install` command supports installing template package from a NuGet package feed. Additionally, you can install a template package from a _.nupkg_ file directly.
+
+> [!IMPORTANT]
+> If you're using .NET 6, the command is `dotnet new --install`.
```
Comment (tdykstra) [inline docs/core/tutorials/cli-templates-create-template-package.md:40]:
```suggestion
> If you're using .NET 6 SDK or earlier, the command is `dotnet new --install`.
```

File: docs/core/tutorials/cli-templates-create-template-package.md [hunk 2]
```diff
@@ -96,7 +99,9 @@ The settings under `<PropertyGroup>` in the XML snippet are broken into three gr
 The first group deals with properties required for a NuGet package. The three `<Package*>` settings have to do with the NuGet package properties to identify your package on a NuGet feed. Specifically the `<PackageId>` value is used to uninstall the template package with a single name instead of a directory path. It can also be used to install the template package from a NuGet feed. The remaining settings, such as `<Title>` and `<PackageTags>`, have to do with metadata displayed on the NuGet feed. For more information about NuGet settings, see [NuGet and MSBuild properties](/nuget/reference/msbuild-targets).
 
 > [!NOTE]
-> To ensure that the template package appears in `dotnet new --search` results, set `<PackageType>` to `Template`.
+> To ensure that the template package appears in `dotnet new search` results, set `<PackageType>` to `Template`.
+>
+> If you're using .NET 6, the command is `dotnet new --search`.
```
Comment (tdykstra) [inline docs/core/tutorials/cli-templates-create-template-package.md:104]:
```suggestion
> If you're using .NET 6 SDK or earlier, the command is `dotnet new --search`.
```

File: docs/core/tutorials/cli-templates-create-template-package.md [hunk 3]
```diff
@@ -142,6 +147,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
 Next, install the template package with the `dotnet new install` command.
 
+> [!IMPORTANT]
+> If you're using .NET 6, the command is `dotnet new --install`.
```
Comment (tdykstra) [inline docs/core/tutorials/cli-templates-create-template-package.md:151]:
```suggestion
> If you're using .NET 6 SDK or earlier, the command is `dotnet new --install`.
```

File: docs/core/tutorials/cli-templates-create-template-package.md [hunk 4]
```diff
@@ -160,6 +168,9 @@ If you uploaded the NuGet package to a NuGet feed, you can use the `dotnet new i
 
 No matter how you installed the template package, either with the _.nupkg_ file directly or by NuGet feed, removing a template package is the same. Use the `<PackageId>` of the template you want to uninstall. You can get a list of templates that are installed by running the `dotnet new uninstall` command.
 
+> [!IMPORTANT]
+> If you're using .NET 6, the command is `dotnet new --uninstall`.
```
Comment (tdykstra) [inline docs/core/tutorials/cli-templates-create-template-package.md:172]:
```suggestion
> If you're using .NET 6 SDK or earlier, the command is `dotnet new --uninstall`.
```

**Review by tdykstra — APPROVED** [review]
Looks good, minor clarification suggestions.

**Review by vlada-shubina — COMMENTED** [review]
I'm ok with this change, however the tutorial has the following prerequisite:
* [.NET SDK 7.0.100](https://dotnet.microsoft.com/download) or a later version.

and using old SDK somewhat contradicts it. 
It might be better to put the note once, as we did in other articles as:
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new#description

**Comment by adegeo** [comment]
I'm OK with just doing a single note at the top instead.

@tdykstra which do you prefer?

**Comment by tdykstra** [comment]
> I'm OK with just doing a single note at the top instead.
> 
> @tdykstra which do you prefer?

Single note at the top.

**Pull request merged**